rotkehlchen/tests/api/test_makerdao_vaults.py

```diff
@@ -891,3 +894,48 @@ def test_query_vaults_usdc_strange(rotkehlchen_api_server, ethereum_accounts):
     details = assert_proper_response_with_result(response)
     expected_details = [vault_7538_details]
     assert_serialized_lists_equal(expected_details, details)
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_modules', [['makerdao_vaults']])
+@pytest.mark.parametrize('mocked_proxies', [{
+    TEST_ADDRESS_1: '0x689D4C2229717f877A644A0aAd742D67E5D0a2FB',
+}])
+@pytest.mark.parametrize('have_decoders', [True])
+def test_delete_vault_owner(rotkehlchen_api_server: 'APIServer'):
+    """Check that deleting owner of a makerdao vault doesnt break anything"""
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    response = requests.put(
+        api_url_for(
+            rotkehlchen_api_server,
+            'blockchainsaccountsresource',
+            blockchain=SupportedBlockchain.ETHEREUM.serialize(),
+        ),
+        json={'accounts': [{'address': TEST_ADDRESS_1}]},
+    )
+    assert_proper_response(response)
+    rotki.chains_aggregator.eth_modules['makerdao_vaults'].get_vaults()  # type: ignore[attr-defined]  # Calling to get data populated  # noqa: E501
+
+    response = requests.delete(
+        api_url_for(
+            rotkehlchen_api_server,
+            'blockchainsaccountsresource',
+            blockchain=SupportedBlockchain.ETHEREUM.serialize(),
+        ),
+        json={'accounts': [TEST_ADDRESS_1]},
+    )
+    assert_proper_response(response)
+
+    response = requests.get(
+        api_url_for(
+            rotkehlchen_api_server,
+            'blockchainbalancesresource',
+            blockchain=SupportedBlockchain.ETHEREUM.serialize(),
+        ),
+        json={
+            'blockchain': SupportedBlockchain.ETHEREUM.serialize(),
```

Comment:
blockchain doesn't need to be in the json too as it is in the argument I believe